netsgiro/utils.py

```diff
@@ -0,0 +1,29 @@
+from contextlib import suppress
+from datetime import date, datetime, timedelta
+from zoneinfo import ZoneInfo
```

Comment:
zoneinfo is only 3.9 and above, so we should probably stick with pytz

Reply:
Good catch. 

There is also `backports.zoneinfo` for 3.6-3.8: https://pypi.org/project/backports.zoneinfo/.

Would you still prefer pytz?

Reply:
I added the backport in a9f11cce2e6de2fc40a371d3ee04a9050712a1ed, but let me know if you want pytz instead and I'll replace it.